app/src/main/java/com/x8bit/bitwarden/data/platform/manager/PushManagerImpl.kt

```diff
@@ -153,7 +153,12 @@ class PushManagerImpl @Inject constructor(
             -> {
                 val payload: NotificationPayload.SyncCipherNotification =
                     json.decodeFromString(notification.payload)
-                if (!isLoggedIn(userId) || !payload.userMatchesNotification(userId)) return
+                if (payload.id == null ||
```

Comment:
I could leave the ID as nonnull and just do this:

```kotlin
                val payload: NotificationPayload.SyncCipherNotification? =
                    json.decodeFromStringOrNull(notification.payload)
                if (payload == null ||
                    !isLoggedIn(userId) ||
                    !payload.userMatchesNotification(userId)
                ) {
                    return
                }
```

Reply:
Yeah that's definitely worth considering as well, although if we know for sure that it's "expected" that some of this data will be missing then its probably more correct to change the nullability of the ID/date.